### ISSUE TITLE
Feature: ability to `bake` with many repos or a `--file` flag with a number of repos

### ISSUE THREAD

**Issue by jpmcb** [open-sauced/pizza-cli#6]
### Type of feature

🍕 Feature

### Current behavior

Currently, multiple calls are required for in order to `bake` multiple repos.

### Suggested solution

We should allow users to provide a list of repos when calling `bake` and define a `--file` flag that has a list of many repos in it to send to the pizza-oven service.

So, for multiple repos on the command line, that would look like:

```
$ pizza bake https://github.com/open-sauced/pizza https://github.com/open-sauced/pizza-cli ... etc.
```

With a file, the file definition might look like:
```yaml
repos:
    - https://github.com/open-sauced/pizza
    - https://github.com/open-sauced/pizza-cli
    - ... etc.
```

And that could be provided by:

```
$ pizza bake --file my-repos.yaml
```

The tricky thing will be getting the UX of this all right but should lead to a slick experience for doing big batch jobs with many repos.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by k1nho** [comment]
I would like to investigate this one! is there an expected behavior for say this case

```bash
$ pizza bake https://github.com/open-sauced/pizza https://github.com/open-sauced/pizza-cli ... --file myrepos.yaml
```

**Comment by jpmcb** [comment]
@k1nho - good question!!!

In the case that the user provides _both_ the file flag and repo URLs, the CLI should parse both. We may run into some odd edge cases where there are overlaps but from a user experience perspective, I can see the need to have a "base" file and add some other repos directly on the CLI.

**Comment by jpmcb** [comment]
Closed by #12
